phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/common/widgets/lists/icons/IconsListElement.java

```diff
@@ -80,13 +81,14 @@ public IconsListElement(E item, URI miniatureUri, String title, boolean enabled)
             this.setClip(clip);
         });
 
-        final Label label = new Label(title);
-        label.getStyleClass().add("miniatureText");
+        final Region miniature = new Region();
```

Comment:
Hmm, the problem is I don't think these URIs belong inside the `Theme` nor `ThemeManager` classes, because they are specific to the list widget.
Maybe we can add them to the `ListWidgetEntry` class, but I'm not sure...